release-schema.json

```diff
@@ -1 +1,35 @@
-{}
+{
+  "properties": {
+    "statistics": {
+      "title": "Statistics",
+      "description": "Statistics about the contracting process.",
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/Statistic"
+      },
+      "minItems": 1,
+      "uniqueItems": true
+    }
+  },
+  "definitions": {
+    "Statistic": {
+      "properties": {
```

Comment:
Aren't we missing a bunch of properties here?

Reply:
We're depending on the bids extension so this is the only extra property. Did I miss something?

Reply:
Oh, right!